integration/cri-o/cri-o.sh

```diff
@@ -69,29 +69,35 @@ IFS=''
 
 # Skip CRI-O tests that currently are not working
 pushd "${crio_repository_path}/test/"
-cp ctr.bats ctr_kata_integration_tests.bats
-for testName in "${!skipCRIOTests[@]}"
-do
-	sed -i '/'${testName}'/a skip \"'${skipCRIOTests[$testName]}'\"' "ctr_kata_integration_tests.bats"
-done
 
-# selectively skip tests depending on the version of cri-o we're testing with
 CRIO_VERSION=$(echo ${PULL_BASE_REF} | cut -d'-' -f 2)
-echo GOT CRIO VERSION $CRIO_VERSION
-if [ "$CRIO_VERSION" != "main" ]; then
-    if [ -z "$CRIO_VERSION" ]; then
-        echo "Unknown version of cri-o - skipping more tests"
-        CRIO_VERSION="1.21"
-    fi
+if [ -z "$CRIO_VERSION" ]; then
+    echo "Unknown version of cri-o - skipping more tests"
+    CRIO_VERSION="1.21"
```

Comment:
Might be worth logging this hard-coded version (by switching the order of these lines) to help with debugging.

Reply:
Thanks - I will add it.
Note that I'm also logging it when editing the test files (the logs will show "skipped for crio vX.Y.Z")